packages/go/schemas/types/core/attachment.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$id": "/core/attachment",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Attachment",
+  "description": "An attachment is a file that is associated with a definition.",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "filename",
+    "content"
+  ],
+  "properties": {
+    "filename": {
+      "type": "string"
+    },
+    "contentType": {
```

Comment:
what's the idea of having both this and AttachmentContentType?

Reply:
I'll rename to "AttachmentContentFormat" - is more accurate